testsuites/syncgateway/functional/tests/test_db_online_offline.py

```diff
@@ -460,9 +461,13 @@ def test_online_to_offline_changes_feed_controlled_close_longpoll(params_from_ba
     with concurrent.futures.ThreadPoolExecutor(max_workers=libraries.testkit.settings.MAX_REQUEST_WORKERS) as executor:
         futures = dict()
         futures[executor.submit(seth.start_longpoll_changes_tracking, termination_doc_id=None)] = "polling"
-        futures[executor.submit(doc_pusher.add_docs, num_docs)] = "docs_push"
+        #  For windows, requiest is too slow, so have _offline request to start before creating docs
+        if sg_platform == "windows":
+            futures[executor.submit(sg_client.take_db_offline, cluster_conf, "db")] = "db_offline_task"
         time.sleep(5)
-        futures[executor.submit(sg_client.take_db_offline, cluster_conf, "db")] = "db_offline_task"
+        futures[executor.submit(doc_pusher.add_docs, num_docs)] = "docs_push"
+        if sg_platform != "windows":
+            futures[executor.submit(sg_client.take_db_offline, cluster_conf, "db")] = "db_offline_task"
```

Comment:
is db_offline_task need to run twice (before and after docs_push? if so, please add comments to explain it does need to be run twice, otherwise, one of them needs to be removed.